src/ResourceManager/Profile/Commands.Profile.Test/EnvironmentCmdletTests.cs

```diff
@@ -262,7 +262,7 @@ public void CanCreateEnvironmentWithAllProperties()
             cmdlet.InvokeEndProcessing();
             Assert.Equal(cmdlet.Name, actual.Name);
             Assert.Equal(cmdlet.EnableAdfsAuthentication.ToBool(), actual.EnableAdfsAuthentication);
-            Assert.Equal(cmdlet.ActiveDirectoryEndpoint, actual.ActiveDirectoryAuthority);
+            Assert.Equal(cmdlet.ActiveDirectoryEndpoint + "/", actual.ActiveDirectoryAuthority, StringComparer.OrdinalIgnoreCase);
```

Comment:
@cormacpayne if we already have a function "EnsureTrailingSlash" that we rely on making sure we have trailing slash? Why in the assert are we adding one?